### PR TITLE
Added that ugrade script now supports appliance FQDN

### LINKED ISSUE #2422: Document that ugrade script now supports appliance FQDN
Per https://github.com/vmware/vic-product/issues/1731

### PR DESCRIPTION
**Pull request by vidya-v**
Hi @stuclem ,

Modified https://vmware.github.io/vic-product/assets/files/html/1.5/vic_vsphere_admin/upgrade_appliance.html to include support for appliance FQDN.
Fixes #2422 
Request your review.

Thanks,
Vidya

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/upgrade_appliance.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ For information about the supported upgrade paths for all versions of vSphere In
 7. Provide information about the old version of the appliance.
 
     1. Enter the name of the datacenter that contains the old version of the appliance.
-    2. Enter the IP address of the old version of the appliance.The upgrade script does not accept FQDN addresses for the old appliance.
+    2. Enter the IP address or FQDN of the old version of the appliance.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/upgrade_appliance.md]: I think that we need to mention that FQDN is only accepted from 1.5.3 onwards...

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/upgrade_appliance.md]: Thanks @stuclem . Made the change.

File: docs/user_doc/vic_vsphere_admin/upgrade_appliance.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ For information about the supported upgrade paths for all versions of vSphere In
 7. Provide information about the old version of the appliance.
 
     1. Enter the name of the datacenter that contains the old version of the appliance.
-    2. Enter the IP address of the old version of the appliance.The upgrade script does not accept FQDN addresses for the old appliance.
+    2. Enter the IP address or FQDN of the old version of the appliance. You can only enter the FQDN for vSphere Integrated Containers 1.5.3 and later versions. For earlier versions, enter the IP address.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/upgrade_appliance.md]: I think that in 1.5.3 and onwards you can enter either the FQDN or the IP address....

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/upgrade_appliance.md]: Thanks, @stuclem . I have updated it.

**Comment by vidya-v** [comment]
Thanks, Stuart!

**Comment by stuclem** [comment]
Per comment from @renmaosheng in https://github.com/vmware/vic-product/issues/1731#issuecomment-496405619, we apparently haven't fixed this in 1.5.3. My mistake! @vidya-v I am reverting this PR following instruction from Steven.

**Pull request merged**